go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/brave/nitriding
 
-go 1.19
+go 1.20
```

Comment:
This isn't the version with opt-out telemetry is it?

Reply:
No. The system hasn't been built yet. Also, they changed it to opt-in.